torch/_torch_docs.py

```diff
@@ -595,10 +595,10 @@ def merge_dicts(*dicts):
            r"""
 atan2(input, other, out=None) -> Tensor
 
-Returns a new tensor with the arctangent of the elements of :attr:`input`
-and :attr:`other`.
+Returns a new tensor with the arctangent of the angle between
+:math:`(\text{{other}}_{{i}}, \text{{input}}_{{i}})` and :math:`(1, 0)`.
```

Comment:
If you talk about vectors here, it should be be `(input[i], other[i])` no?
Or you can define it as `tan^{-1} ( other[i] / input[i])`.

Reply:
Actually maybe something inspired from what [the numpy doc](https://docs.scipy.org/doc/numpy/reference/generated/numpy.arctan2.html) says would be clearer?

Reply:
I rewrote this paragraph. Please see the new update